client/src/components/JobTitleSelect/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Dropdown } from 'semantic-ui-react';
+import styles from './styles.module.scss';
+import { UserProfileState } from 'containers/ProfilePage/logiс/state';
+
+interface Props {
+	text: string;
+	propKey: keyof UserProfileState;
+	placeholder: string;
+	title: string;
+	handleSelect: (propKey: keyof UserProfileState, value: string) => void;
+}
+
+const options = [
+	{ key: 'repair', text: 'Database administrator', value: 'Database administrator' },
+	{ key: 'ruby', text: 'Back-end developer', value: 'Back-end developer' },
+	{ key: 'ui', text: 'Front-end developer', value: 'Front-end developer' },
+	{ key: 'ux', text: 'Full-Stack developer', value: 'Full-Stack developer' },
+];
```

Comment:
1. Are these options hardcoded on client side? 
2. What if we want to extend them? 
3. Can we implement it on server using enum with possible values? As it was done for board type ([link](https://github.com/BinaryStudioAcademy/bsa-2020-TaskyHusky/blob/dev/server/src/models/Board.ts))